ios/ReactNativePayments.m

```diff
@@ -186,24 +187,24 @@ - (void) paymentAuthorizationViewController:(PKPaymentAuthorizationViewControlle
                    didSelectShippingContact:(PKContact *)contact
                                  completion:(nonnull void (^)(PKPaymentAuthorizationStatus, NSArray<PKShippingMethod *> * _Nonnull, NSArray<PKPaymentSummaryItem *> * _Nonnull))completion
 {
-    self.shippingContactCompletion = completion;
-
-    CNPostalAddress *postalAddress = contact.postalAddress;
-    // street, subAdministrativeArea, and subLocality are supressed for privacy
-    [self.bridge.eventDispatcher sendDeviceEventWithName:@"NativePayments:onshippingaddresschange"
-                                                    body:@{
-                                                           @"recipient": [NSNull null],
-                                                           @"organization": [NSNull null],
-                                                           @"addressLine": [NSNull null],
-                                                           @"city": postalAddress.city,
-                                                           @"region": postalAddress.state,
-                                                           @"country": [postalAddress.ISOCountryCode uppercaseString],
-                                                           @"postalCode": postalAddress.postalCode,
-                                                           @"phone": [NSNull null],
-                                                           @"languageCode": [NSNull null],
-                                                           @"sortingCode": [NSNull null],
-                                                           @"dependentLocality": [NSNull null]
-                                                           }];
+    // self.shippingContactCompletion = completion;
```

Comment:
instead of commenting out, see if we can just gate it behind a check to see if requestShipping option is true.